model_training/local_prediction/train_test_split/test.txt

```diff
@@ -0,0 +1,42 @@
+data/obj/34.jpg
```

Comment:
Should remove this + `train.txt` since the data folder is not publicly available. I assume these files can be generated by a script given an image folder?

Reply:
@yarkhinephyo The train val split is random every time, how do you want to keep track of this?

Reply:
You can just keep in ur gdrive for now. The images won't be repo anyway aside from a few samples so that they can replicate the labeling and training